contrib/inventory/ec2.py

```diff
@@ -881,6 +882,16 @@ def add_instance(self, instance, region):
             if self.nested_groups:
                 self.push_group(self.inventory, 'instance_states', state_name)
 
+        # Inventory: Group by platform
+        if self.group_by_platform:
+            if instance.platform:
+                platform = self.to_safe('platform_' + instance.platform)
+            else:
+                platform = self.to_safe('platform_undefined')
+            self.push(self.inventory, platform, dest)
```

Comment:
I'm going to add a commit to change this to hostname. hostname defaults to dest if it isn't defined, so I believe the behavior will stay the same and also then maintain consistency with the surrounding code.